src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page-body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__code">Product code: 195434</div>
+      <div class="card__review">
+        <div class="stars">
+          <div class="stars__group">
+            <div class="stars__group-star"></div>
+            <div class="stars__group-star"></div>
+            <div class="stars__group-star"></div>
+            <div class="stars__group-star"></div>
+            <div class="stars__group-star"></div>
+          </div>
+          <p class="stars__text">Reviews: 5</p>
+        </div>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">Price:</p>
+        <p class="card__price-sum">$2,199</p>
+      </div>
+      <button
+        class="card__buy"
+        data-qa="hover"
+      >
+        <span class="card__buy-text">BUY</span>
+      </button>
```

Comment:
The buy button lacks an `aria-label` attribute. This attribute is important for screen readers and should describe the action that will happen when the button is clicked. For example, `aria-label="Buy APPLE A1419 iMac 27" Retina 5K Monoblock"`.